lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -288,13 +288,6 @@ sub restore_libvirt_default {
     }
 }
 
-sub enable_libvirt_log {
-    assert_script_run qq(echo 'log_level = 1
-    log_filters="3:remote 4:event 3:json 3:rpc"
-    log_outputs="1:file:/var/log/libvirt/libvirtd.log"' >> /etc/libvirt/libvirtd.conf);
-    restart_libvirtd;
```

Comment:
This libvirtd restart is to load the new config file.

Reply:
I remove the entire function because I replace it with the common function turn_on_libvirt_debugging_log() in libvirt_virtual_network_init.pm.